usr/share/rear/layout/recreate/default/200_run_layout_code.sh

```diff
@@ -20,6 +20,30 @@
 #   fi
 #
 
+function lsblk_output () {
```

Comment:
There is another place where `lsblk` output is shown for user information https://github.com/rear/rear/blob/42d0ef8c32afb1809190bfc52c193bd3fa77a93a/usr/share/rear/layout/save/GNU/Linux/100_create_layout_file.sh#L35 .
Could the two be unified? There are tiny differences in the list of columns between the tw, are they intentional?

Reply:
Sorry, I se in the comment that some of them are intentional. Still, here we have MOUNTPOINTS, in 100_create_layout_file.sh we have MOUNTPOINT. Is that intentional?

Reply:
Not sure what would be the ideal solution here. If the code is not shared, it will accumulate tiny and unintentional differences. On the other hand, there are many intentional differences that would make any shared version quite ugly.

Reply:
The currently local function lsblk_output()
is currently intentionally a local function.
I know the other places where lsblk output is shown.
This will be cleaned up later via a separated pull request.
I cannot and want not to do all that at once.
For now I like to see how it behaves with the changes
in this pull request in practice.
We are in ReaR 2.8 development phase where things can be
a bit inconsistent.
For the ReaR 2.8 release those things should be cleaned up.

E.g. also the currently undocumentes config variables
about 'barrel' support. I cannot know now what the right way
will be so I don't want to have them in default.conf until
I got a better gut feeling how things should be in practice.